fs/xfstests.py.data/README

```diff
@@ -0,0 +1,43 @@
+This is a simple wrapper for running xfstests inside avocado. The steps to get
+started are really simple:
+
+1) Edit the configuration variables on the yaml file as well environment variable
```

Comment:
@apahim while start the test  env variable need to set as "TESTDIR and TESTDEV using export"  and configuration value will  be like test_number and skip_dangerous